cookbooks/aws-parallelcluster-config/recipes/update.rb

```diff
@@ -21,8 +21,14 @@
   end
 end
 
+ruby_block "backup storages mappings" do
```

Comment:
I'd say explicitly in a comment on in the resource description that we are backing up these settings becase they need to be used for a potential rollback

Reply:
Added ac comment, it is also used for update